### DISCUSSION TITLE
Question on tracking?

### DISCUSSION THREAD

**Discussion by killmasta93** [postalserver/postal#3095] [Help with using Postal] [answered]
HI
I was wondering if someone could shed some light i was re reading the wiki guide https://docs.postalserver.io/features/click-and-open-tracking
and filtering in the discussion sections, which few discussions mention about having another IP, other discussions talked about checking the URL to see if it works and im a bit confused 

So these are the steps i took from what i understood,

on the webGUI tracking domains created domain click.domain.co
also created a CNAME pointing to  track.postal.domain.co

![image](https://github.com/user-attachments/assets/eed76622-3317-4cee-88e5-70cefbb94e9b)

im a bit confused on the DNS part not sure if i understood clearly this is what i configured on Cloudflare DNS

![image](https://github.com/user-attachments/assets/22651a1b-b72e-4ce0-a89d-be19732ca052)

and i assume the click.domain.co when i go to the URL should show the hello page? 

Thank you

**Comment by willpower232** [comment]
Sounds like you've fully understood it. The hello page just confirms everything is working. Have you tried sending a message yet to see the replaced links and link tracking pixel?

**Reply by killmasta93** [reply]
hi @willpower232 Thank you so much for the reply, so forgot to mention using a nginx reverse proxy when i go to click.domain.co im getting http error 403 ERR_HTTP_RESPONSE_CODE_FAILURE not sure if its an issue of nginx or the postal service?

**Reply by willpower232** [reply]
Did you add the click tracking header in your nginx config for click.domain.co?

**Reply by killmasta93** [reply]
yeah currently running bunkerweb and added it to the headers 

![image](https://github.com/user-attachments/assets/377c05e3-3629-42b4-b73a-d4a82e0489d0)
is there another way i can check if its a postal issue or an nginx issue?

**Reply by killmasta93** [reply]
hi @willpower232 so i put a default nginx proxy added this but still getting 403 issue
```

    location / {
        proxy_set_header X-Real-IP  $remote_addr;
        proxy_set_header Host $host;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto https;
        proxy_set_header X-Postal-Track-Host: 1;
set $upstream_app 192.168.7.106;
        set $upstream_port 5000;
        set $upstream_proto http;
        proxy_pass $upstream_proto://$upstream_app:$upstream_port;

```
on the postal logs i see this

`postal-web-1     | 190.xx.xxx - - [25/Aug/2024:18:55:51 +0000] "GET / HTTP/1.0" 403 - 0.0014`

any idea what i missed?

**Reply by killmasta93** [reply, marked as answer]
it seems i had a typo the working config is 

```
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
       proxy_set_header Host $http_host;
       proxy_set_header  X-Forwarded-Proto $scheme;
       proxy_set_header  X-Forwarded-Ssl on;
       proxy_set_header  X-Forwarded-Port $server_port;
       proxy_set_header  X-Forwarded-Host $host;
proxy_set_header X-Postal-Track-Host 1;
```

**Comment by plancked** [comment]
I have the following question:

since click.domain.com is CNAME to track.postal.mypostaldomain.com, when why should I proxy web traffic from one to the other?
